hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/server/JsonUtils.java

```diff
@@ -79,14 +81,18 @@ public static ObjectNode createObjectNode(Object next) {
     return MAPPER.valueToTree(next);
   }
 
+  public static JsonNode valueToJsonNode(Object value) {
+    return MAPPER.valueToTree(value);
+  }
+
   public static JsonNode readTree(String content) throws IOException {
     return MAPPER.readTree(content);
   }
 
-  public static List<HashMap<String, Object>> readTreeAsListOfMaps(String json)
+  public static ArrayList<LinkedHashMap<String, Object>> readTreeAsListOfMaps(String json)
```

Comment:
We should replace `ArrayList` with `List` with `ArrayList` but the other way.

BTW, this method is only used in tests.  Create a `JsonTestUtils` in `src/test` and then move the test methods there..